src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/data/AltarStructureMaterialProvider.java

```diff
@@ -51,23 +52,51 @@ public void run(HashCache pCache) throws IOException {
 
     @Override
     public String getName() {
-        return "StructureMaterialProvider["+this.namespace+"]";
+        return "Altar Structure Materials";
```

Comment:
The name should be different for each instance with a different output namespace

Reply:
Why? Our vanilla providers don't do that either

Reply:
that is the point it is not our provider it is our api for other providers vanilla/forge has them abstract